src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.js

```diff
@@ -118,8 +118,16 @@ class ActivityFeed extends React.Component<Props, State> {
         return feedItems.length === 0 || (feedItems.length === 1 && feedItems[0].type === ItemTypes.fileVersion);
     };
 
-    hasLoaded = ({ currentUser, feedItems }: Props = this.props): boolean =>
-        currentUser !== undefined && feedItems !== undefined;
+    /**
+     * Determines whether currentUser and feedItems have loaded.
+     * @param prevCurrentUser - The previous value of the currentUser prop
+     * @param prevFeedItems - The previous value of the feedItems prop
+     * @returns {boolean}
+     */
+    hasLoaded = (prevCurrentUser?: User, prevFeedItems?: FeedItems): boolean => {
```

Comment:
This changes the logic to be more of a `hasJustRecentlyLoaded()` (as in, in the previous cycle `currentUser` and `feedItems` wasn't set, but now it is or isn't), which if I recall we moved away from to simplify other logic. Does this change still work with the loading state in the `render()`? Just double checking, because we ran into a lot of problems with getting this logic working.

Reply:
@tjgupta The loading indicator described in this block shows up while the ActivityFeed is loading items:
```
{isLoading && (
   <div className="bcs-activity-feed-loading-state">
       <LoadingIndicator />
   </div>
)}
```
Is there anything else I should check?